eng/versioning/version_client.txt

```diff
@@ -496,6 +496,7 @@ io.clientcore:annotation-processor-test;1.0.0-beta.1;1.0.0-beta.1
 
 unreleased_com.azure:azure-json;1.5.0-beta.1
 unreleased_io.clientcore:core;1.0.0-beta.6
+unreleased_io.clientcore:annotation-processor;1.0.0-beta.1
```

Comment:
This doesn't make sense to me as this unreleased is the same as the dependency version as this has never shipped.